config/gazebo9_ubuntu_bionic.ppa.yaml

```diff
@@ -0,0 +1,8 @@
+# include the fix for server_fixture while Ubuntu merges the patch:
+# https://bugs.launchpad.net/ubuntu/+source/gazebo/+bug/1773811
+name: gazebo9_import_ubuntu_bionic
+method: http://ppa.launchpad.net/j-rivero/gazebo9bionic/ubuntu
+suites: [bionic]
+component: main
+architectures: [amd64, i386, armhf, arm64, source]
+filter_formula: Package (% *gazebo* )
```

Comment:
The other rules seem to be more specific on the filter regex: https://github.com/ros-infrastructure/reprepro-updater/blob/2f3e9b59aa53601fe32b06b9bb8f6ddbb71c4790/config/gazebo9_ubuntu_artful.upstream.yaml#L6
Can we adapt this one to explicitly list the packages as well ?